### ISSUE TITLE
Success message

### ISSUE THREAD

**Issue by koterpillar** [tweag/FawltyDeps#222]
**Is your feature request related to a problem? Please describe.**

When/after the project has no problems FawltyDeps detects, the output is:

```
$ poetry run fawltydeps        

For a more verbose report re-run with the `--detailed` option.
```

This is slightly confusing as I would expect _either_:
* A hardcore behavior of only setting the exit code and not printing anything (cf. `rm important_data` or `grep hello /dev/null`)
* A status message like "No unused or undeclared dependencies detected."

**Describe the solution you'd like**

When there are no problems, output "No unused or undeclared dependencies detected." before suggesting `--detailed`.

**Comment by jherland** [comment]
Submitted #224 to fix this. Thanks for reporting!

**Comment by jherland** [comment]
I just released v0.7.0 to PyPI, where this is fixed.
